packages/react-google-maps-api/src/LoadScript.tsx

```diff
@@ -179,27 +174,9 @@ class LoadScript extends React.PureComponent<LoadScriptProps, LoadScriptState> {
       preventGoogleFonts()
     }
 
-    const params = [`key=${this.props.googleMapsApiKey}`]
-
-    if (this.props.version) {
-      params.push(`v=${this.props.version}`)
-    }
-
-    if (this.props.language) {
-      params.push(`language=${this.props.language}`)
-    }
-
-    if (this.props.region) {
-      params.push(`region=${this.props.region}`)
-    }
-
-    if (this.props.libraries && this.props.libraries.length) {
-      params.push(`&libraries=${this.props.libraries.join(",")}`)
-    }
-
     const injectScriptOptions = {
-      id: this.props.id,
-      url: `https://maps.googleapis.com/maps/api/js?${params.join('&')}`
+      id: this.props.id!,
```

Comment:
why `!` ?

Reply:
Because of stupid class & defaultProps :) TypeScript is not able to see that default value is assigned so it will never be undefined.

Reply:
For the reference, it was required a prop before, but since it has default, there is no point in requiring it.

Reply:
@FredyC 
I have issues with running gatsby-example locally after yarn workspaces merge, can you try to run it, you will see what I mean.